connector/kafka-0-10/src/test/scala/org/apache/spark/streaming/kafka010/KafkaRDDSuite.scala

```diff
@@ -117,7 +118,7 @@ class KafkaRDDSuite extends SparkFunSuite {
     )
     messages.foreach { case (k, v) =>
       val record = new SimpleRecord(k.getBytes, v.getBytes)
-      log.appendAsLeader(MemoryRecords.withRecords(CompressionType.NONE, record), 0);
+      log.appendAsLeader(MemoryRecords.withRecords(Compression.NONE, Seq(record): _*), 0);
```

Comment:
This is a new feature of Kafka 3.8.0 (KAFKA-7632, KIP-390).
- https://github.com/apache/kafka/pull/15516